bundles/CoreBundle/src/Command/SearchBackendReindexCommand.php

```diff
@@ -34,6 +35,13 @@ protected function configure()
         $this
             ->setName('pimcore:search-backend-reindex')
             ->setAliases(['search-backend-reindex'])
+            ->addOption(
+                'full',
+                'f',
+                InputOption::VALUE_OPTIONAL,
```

Comment:
Wouldn't `VALUE_NONE` be better? The CLI call then would look like `bin/console pimcore:search-backend-reindex --full`.